.circleci/config.yml

```diff
@@ -69,7 +69,13 @@ jobs:
     working_directory: ~/repo
     steps:
       - checkout
-      - run: make -j$(nproc)
+      - run: echo deb http://deb.debian.org/debian stretch-backports main > /etc/apt/sources.list.d/debian-backports.list
+      - run: apt-get update && apt-get install -y -t stretch-backports cmake
+      # Install cmake files (https://github.com/devkitPro/docker/issues/3)
+      - run: dkp-pacman -S --needed --noconfirm --quiet devkitpro-pkgbuild-helpers
+      - run: mkdir build
+      - run: cd build && cmake .. -DBINARY_RELEASE=ON -DNONET=ON -DCMAKE_TOOLCHAIN_FILE=../CMake/switch/devkita64-libnx.cmake
+      - run: cd build && make -j$(nproc)
       - store_artifacts: {path: ./release/devilutionx.nro, destination: devilutionx.nro}
```

Comment:
No artifact files found at /root/repo/release/devilutionx.nro

Reply:
Ah, right, they're placed in build now

Reply:
Should be fixed now